teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Scoring functions are used to <strong>discriminate</strong></li><li><strong>Force field </strong><em>// (written in two words, no hyphen)</em></li><li><strong>protein-ligand </strong>complex</li><li>Emp<strong>i</strong>rical scoring functions are based on coefficients that are used <strong>to estimate </strong>the contributions of different interaction types<strong>; e.g.</strong> hydrogen bonds, ionic interaction<strong>s</strong>, hydrophobic contacts.</li><li>experimentally resolved <strong>protein-ligand</strong> complexes</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>